app/src/main/java/com/codesoom/assignment/util/JsonUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.codesoom.assignment.util;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class JsonUtil {
+
+    private static ObjectMapper objectMapper = new ObjectMapper();
+
+    /**
+     * JsonString으로 변환한다.

```

Comment:
JavaDoc은 리턴값을 설명하는 것이 중요합니다.
```suggestion
     * Json 형식의 문자열로 변환해 리턴합니다.
```

Reply:
Input, Ouput에 대한 설명을 빼먹다니..
저의 나태함에 반성합니다 ㅜㅜ

return값에 대한 설명을 적어놓은 버번을 Push 해놓았습니다.

JavaDoc에 대해 구글링을 해봤는데
"@return" 이라는 어노테이션이 있더군요.

있는것 쓰는게 좋을것같아서 반환값이 있는 메서드에는 
해당 어노테이션을 써서 리턴값에 대해 적어 보았습니다!

Reply:
그건 어노테이션이라고 부르지 않고 JavaDoc 태그라 부릅니다. 생긴건 `@` 때문에 비슷해 보이지만 엄연히 달라요. 이번 기회에 검색해서 알아보시는 것도 좋을 거에요.